lib/bx/align/_epo.pyx

```diff
@@ -165,9 +165,9 @@ cpdef numpy.ndarray[numpy.uint64_t, ndim=2] bed_union( numpy.ndarray[numpy.uint6
 
 #@cython.wraparound(False)
 #@cython.boundscheck(False)
-cpdef numpy.ndarray[numpy.int64_t, ndim=2] cummulative_intervals(numpy.ndarray[numpy.int64_t, ndim=1] S,
+cpdef numpy.ndarray[numpy.int64_t, ndim=2] cumulative_intervals(numpy.ndarray[numpy.int64_t, ndim=1] S,
```

Comment:
I think this breaks the library API, is it worth? Should we provide an alias?

Reply:
Then I'd vote for fixing the typo and adding an alias for backward compatibility.